### PR TITLE
[Bug] JVM parameters distinguish between client and server

### PR DESCRIPTION
**Pull request by MonsterChenzhuo**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by MonsterChenzhuo** [comment]
@EricJoy2048  I made a simple distinction so that there would be two configuration files under config, and starting the client or server would read the respective jvm configuration files. I'm not sure if it's appropriate.

**Review by EricJoy2048 — DISMISSED** [review]

**Comment by TyrantLucifer** [comment]
Offer the test snapshot images to verify you change has worked. You can compile the whole project and change the different jvm parameter between client and server.

**Comment by MonsterChenzhuo** [comment]
> Offer the test snapshot images to verify you change has worked. You can compile the whole project and change the different jvm parameter between client and server.

The historical e2e is enough to cover the current changes.
org.apache.seatunnel.core.starter.seatunnel.jvm.JvmOptionsParserTests

**Review by TaoZex — DISMISSED** [review]
+1

**Comment by MonsterChenzhuo** [comment]
图片
图片
cc @TyrantLucifer

**Comment by Hisoka-X** [comment]
Hi, please add default jvm heap size with 128M for client.

**Comment by MonsterChenzhuo** [comment]
@TyrantLucifer @Hisoka-X @EricJoy2048  Thanks for your review,I have made the changes.

File: config/jvm_client_options [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## -Xmx4g
+##
+##
+################################################################
+
+# Xms represents the initial size of total heap space
+# Xmx represents the maximum size of total heap space
+
+-Xms128m
+-Xmx1g
```
Comment (Hisoka-X) [inline config/jvm_client_options]:
```suggestion
-Xmx512m
```

**Review by Hisoka-X — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**